### DISCUSSION TITLE
Load and save dynamic parameters

### DISCUSSION THREAD

**Discussion by hmueller01** [khoih-prog/ESPAsync_WiFiManager_Lite#17] [Q&A] [answered]
Would it be ok, if I make `loadDynamicData()` and `saveDynamicData()`_public_?
I'd like to change some of the dynamic parameters during runtime and save the values back to FS ...

`loadDynamicData()` because I like to use the parameters before `begin()`.

**Comment by khoih-prog** [comment]
Certainly you can. Just testing thoroughly to know everything is still OK.

It's good if you can share your experience here then, good or bad.

Good Luck,

**Comment by hmueller01** [comment]
See PR #20

**Comment by hmueller01** [comment, marked as answer]
Implemented by PR #24.
